pages/api/check-status.ts

```diff
@@ -63,24 +68,27 @@ export const searchPassportStatusApi = async (
   const response = await fetch(url)
 
   if (response.ok) {
+    logger.info('response OK')
     const searchResult: PassportStatusesSearchResult = await response.json()
     const { GetCertificateApplicationResponse } = searchResult._embedded
 
     if (GetCertificateApplicationResponse.length === 0) {
-      res.status(404).send('Passport Status Not Found')
-      return
+      logger.error('error 404: searchResult._embedded is empty')
+      return res.status(404).send('Passport Status Not Found')
     }
 
-    res.send(mapToCheckStatusApiResponse(GetCertificateApplicationResponse[0]))
-    return
+    return res.send(
+      mapToCheckStatusApiResponse(GetCertificateApplicationResponse[0])
+    )
   }
 
   if (response.status === 404 || response.status === 422) {
-    res.status(404).send('Passport Status Not Found')
-    return
+    logger.error(`error ${response.status}: ${response.body}`)
+    return res.status(response.status).send('Passport Status Not Found')
   }
 
-  res.status(response.status).send(response.statusText)
+  logger.info(`Status: ${response.status}: ${response.statusText}`)
```

Comment:
I would log this as `debug` because this isn't an application error, and we're going to have potentially millions of requests.